apps/frontend/core-dapp/src/stores/entities/FeeEntity.ts

```diff
@@ -42,7 +43,7 @@ export class FeeStore {
   }: {
     applicableToWstEth: () => BigNumber | undefined
     fee: Fraction
-    tokenSender: TokenSenderEntity | undefined
+    tokenSender: TokenSenderEntity | TokenSenderCallerEntity | undefined
```

Comment:
So based on discussion, an idea is to make this hook instead of tokenSender, which only requires the prop to meet requirement of 1 interface, something like this

```
interface FeeHook { 
  calculateReward: (feeInWstEth: BigNumber) => BigNumber
  calculateRewardValueInEth: (ppo: BigNumber) => BigNumber
}
```

```suggestion
    hook: FeeHook
```